spring-kafka/src/main/java/org/springframework/kafka/core/KafkaOperations.java

```diff
@@ -163,6 +167,15 @@
 	void flush();
 
 	/**
+	 * Send a Flux of SenderRecords.
+	 * @param flux the flux.
+	 * @param <T> correlation metadata type.
+	 * @param t the class for the correlation metadata
+	 * @return a flux of SenderResults.
+	 */
+	<T> Flux<SenderResult<T>> send(Flux<SenderRecord<K, V, T>> flux, Class<T> t);
```

Comment:
Do we want to introduce a hard dependency in Spring Kafka on Reactor or should we provide this in a separate interface (e.g. `ReactiveKafkaOperations`).